src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -226,7 +226,7 @@ def build_summary_data(
                     )
 
             for activity_type, groups in deduped_groups_by_activity_type.items():
-                for group in list(groups)[:4]:
+                for group in list(groups)[:3]:
```

Comment:
what does 3 correspond with here? Is this an arbirtrary number?

Reply:
it's the first 3 in the list, for space reasons we need to limit what we show